sdk/include/wdf/kmdf/1.9/wdffuncenum.h

```diff
@@ -0,0 +1,430 @@
+/*
+ * wdffuncenum.h
+ *
+ * Windows Driver Framework - enum of all WDF API Functions
+ *
+ * This file is part of the ReactOS wdf package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDFFUNCENUM_H_
+#define _WDFFUNCENUM_H_
+
+extern PWDF_DRIVER_GLOBALS WdfDriverGlobals;
+
+typedef enum __WDFFUNCENUM
+{
+    WdfChildListCreateTableIndex = 0,
```

Comment:
So, any bugs in this one are not gonna be caught in review because it's just too big. One thing you can do to test it is do some regex replace to get a C source file that looks like this:
```
C_ASSERT(WdfChildListCreateTableIndex == 0);
C_ASSERT(WdfChildListGetDeviceTableIndex == 1);
...
```

Then you can compile the source file against both MS's and our header and you should get errors for any typos.